backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc

```diff
@@ -18,9 +18,17 @@ You configure AWS for Velero, create a default `Secret`, and then install the Da
 To install the OADP Operator in a restricted network environment, you must first disable the default OperatorHub sources and mirror the Operator catalog. See xref:../../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks] for details.
 
 include::modules/oadp-installing-operator.adoc[leveloffset=+1]
+
 include::modules/migration-configuring-aws-s3.adoc[leveloffset=+1]
+
+include::modules/oadp-preparing-aws-credentials.adoc[leveloffset=+1]
+
+include::modules/oadp-installing-oadp-rosa-sts.adoc[leveloffset=+1]
```

Comment:
@HagayVider1 - I do not think this order of modules makes much sense.

Please ensure you understand the difference between installing the OpenShift API for Data Protection (OADP) with Amazon Web Services (AWS) and Red Hat OpenShift® Service on AWS (ROSA). 

Let us talk about this on Thursday and see if we can improve it as people might fall between the options.

Reply:
Accepted. I'll change this.

Reply:
Let us walk through it later today